### PR TITLE
Adding balance_available to the wallet

### PR DESCRIPTION
**Pull request by jsphon**
### Description:
Adding `available_balance` to the Wallet.

### Breaking changes:
- [ ]

### New features:

Available_balance is available in the Wallet.

### Fixes:
- [ ]

### PR status:
- [X] Version bumped
- [X] Change-log updated

### CODE REVIEW

**Review by fintzd — CHANGES_REQUESTED** [review]
This would be a great addition I think, but there is one more file that I could think of right away which would need to be edited accordingly. Could you check it out and add the changes to this pull request?

The file is the one for the REST api: `bfxapi/rest/bfx_rest.py`

Specificall the `get_wallets(self)` function on row 378. 

[Here is the link to the file](https://github.com/bitfinexcom/bitfinex-api-py/blob/4b4a17ebd8f5e9232eb116202370e05914887219/bfxapi/rest/bfx_rest.py)

**Comment by jsphon** [comment]
Thanks for  looking at this PR.

I have made the requested change, and verified that it works using the script in `examples/rest/get_authenticated_data.py`.

**Comment by teresaejunior** [comment]
Have a loot at https://github.com/bitfinexcom/bitfinex-api-py/pull/76

**Comment by jsphon** [comment]
@teresaejunior Thanks. That looks the same as my PR. Do you know why it wasn't merged?

**Comment by itsdeka** [comment]
Thanks a lot for the help @teresaejunior , @jsphon, @fintzd. I am testing it, if I do not see anything wrong I'll ask to merge.

**Comment by itsdeka** [comment]
Looks good. I'd like more to see available_balance instead of balance_available but is not a big deal.

@robertkowalski can you please take a look when possible?

**Review by fintzd — APPROVED** [review]
Looks good to me too! Thanks!
And thank you @itsdeka for checking it out too!

File: bfxapi/websockets/wallet_manager.py [hunk 1]
```diff
@@ -17,13 +17,13 @@ def _update_from_snapshot(self, raw_ws_data):
         wData = raw_ws_data[2]
         self.wallets = {}
         for wallet in wData:
-            new_wallet = Wallet(wallet[0], wallet[1], wallet[2], wallet[3])
+            new_wallet = Wallet(wallet[0], wallet[1], wallet[2], wallet[3], wallet[4])
```
Comment (robertkowalski) [inline bfxapi/websockets/wallet_manager.py:20]: snapshot is not updating correctly for me:

```
[BfxWebsocket] [DEBUG] [0,"ws",[["exchange","AAA",40200.344803994,0,null,null,null],
["exchange","BBB",50202.89552318822,0,null,"Trading fees for 0.50114911 AAA (AAABBB) @ 
16767.0 on BFX (0.2%)",null],["exchange","TESTBTC",1.01996,0,null,"Trading fees for 0.01 
TESTBTC (TESTBTC:TESTUSD) @ 49344.0 on BFX (0.2%)",null],["exchange","TESTUSD",
12.94,0,null,"Exchange 0.01 TESTBTC for TESTUSD @ 49344.0",{"reason":"TRADE","order_id":59935824154,"order_id_oppo":59950486657,"trade_price":"49344.0",
"trade_amount":"0.01"}],["exchange","TESTUSDT",1000,0,null,null,null]]]


Wallet <'exchange_AAA' balance='40200.344803994' balance_available='None' unsettled='0'>
Wallet <'exchange_BBB' balance='50202.89552318822' balance_available='None' unsettled='0'>
Wallet <'exchange_TESTBTC' balance='1.01996' balance_available='None' unsettled='0'>
Wallet <'exchange_TESTUSD' balance='12.94' balance_available='None' unsettled='0'>
Wallet <'exchange_TESTUSDT' balance='1000' balance_available='None' unsettled='0'>
```

Comment (robertkowalski) [reply bfxapi/websockets/wallet_manager.py:20]: ah, initially the available balance is not known

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**